static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -179,7 +179,7 @@ define([
         events.addPrerollEvents(player, mediaId, mediaType);
         events.bindGoogleAnalyticsEvents(player, gaEventLabel);
 
-        videoMetadata.getVideoInfo(el).then(function(videoInfo) {
+        videoMetadata.getVideoInfo($el).then(function(videoInfo) {
```

Comment:
`$el` is [declared above](https://github.com/guardian/frontend/pull/14929/files#diff-b226c7adcb4e3819c2d7fd8c6a997f8dL151)